ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/views/PromotionalOfferView.kt

```diff
@@ -33,15 +36,22 @@ import com.revenuecat.purchases.models.toRecurrenceMode
 import com.revenuecat.purchases.ui.revenuecatui.composables.AppIcon
 import com.revenuecat.purchases.ui.revenuecatui.customercenter.data.CustomerCenterConfigTestData
 import com.revenuecat.purchases.ui.revenuecatui.customercenter.data.PromotionalOfferData
+import com.revenuecat.purchases.ui.revenuecatui.customercenter.data.getColorForTheme
 
 @JvmSynthetic
 @Composable
 internal fun PromotionalOfferView(
```

Comment:
Not from this PR but I wonder if this should be called `PromotionalOfferScreen`? FWIW, it's not recommended to call composables "Views" since they are not actually views. We do call them views in Paywall V2, but it's mostly a name collision issue, since the name would collide with the original "data" components (as in "TextComponent" is the data holder for "TextComponentView"). FFTI though, since it's not part of this PR!